driver/src/main/java/org/neo4j/driver/Config.java

```diff
@@ -304,15 +315,45 @@ public long fetchSize() {
 
     /**
      * Returns notification config.
+     *
      * @return the notification config
      * @since 5.7
+     * @deprecated superseded by {@link #minimumNotificationSeverity()} and
+     * {@link #disabledNotificationClassifications()}
      */
+    @Deprecated
```

Comment:
We should not deprecate the old methods since the new ones still in preview. The deprecation should be done when we move the new methods to GA